docs/source/release-notes/2020-3-3-Demeter.md

```diff
@@ -0,0 +1,127 @@
+            * "Activate/Deactivate Snap to Grid" can be triggered by pressing "H".
+            * "Align Grid to Selected Entities" can be triggered by pressing "J".
+            * "Align Grid to Avatar" can be triggered by pressing "K".
+        * The actions: "Align Grid to Selected Entities" and "Align Grid to Avatar" will turn on the grid if it was not visible. (Doing this implies that you want to have the grid be visible). But you will still need to "Toggle Grid" (G) to turn the grid off. Note that "Align Grid to Selected Entities" will work only if there is at least one entity selected.
+    * Main "Edit" menu:
+        * The two "Import Entities" actions have been brought back to the main "Edit" menu. (Since there was no unanimity about their removal.)
+        * A separator has been added to properly group the 'Create' app's menu items.
+    * In desktop mode only, the default window width is now 750px instead of 490px.
+* App: 'Create' app material icons update. (#907)
+    * Preference: "Show Zones in Create Mode" has been removed. The method behind it is still in the API, but does nothing. It has been abandoned since it was only drawing a box that wasn't useful to assist in figuring out the shape of the zone.
+    * Preference: "Show Lights and Particle Systems in Create Mode" has been renamed to "Show Icons in Create Mode" since this setting was also managing all the icons (including Zones and now Parented Materials). The objective of the setting is mainly to simplify the display in edit mode when a scene is very dense.
+    * `image3D` overlays used for the icons have been migrated to local entities.
+    * All the entity icons (in-world non physical entities) have been updated to be more uniform. They now all have the same dark gray tone, with a white border to keep them visible under different lighting.
+
+#### Miscellaneous
+
+* Misc: Module updates for screenshare app. (#708, #905)
+* Application: Upgrade tbb (oneAPI Threading Building Blocks) to fix a startup crash on Fedora 32 and later. (#863)
+* Build: GHA now builds PR builds with crash reporting enabled on Windows. (#878)
+* Application: The `--forceCrashReporting` command-line parameter has been added to enable crash reporting even if the settings have it off. (#879)
+* Build: Fix hifi_qt.py reporting Ubuntu 20.04 as outdated. (#891)
+* Misc: Update various references for GitHub links in the codebase. (#906)
+
+## Domain Server
+
+The numbers at the end of each item are the PR numbers in the Vircadia [repo](https://github.com/vircadia/vircadia).
+
+#### General
+
+* Server: The Domain server now sends its IP address and port to the Metaverse server with every heartbeat. (#877)
+
+## Metaverse Server & Dashboard (Codename Iamus)
+
+### Iamus [Metaverse Server](https://github.com/vircadia/iamus)
+
+#### Version 2.3.6
+
+<ul>
+<li><a href="http://github.com/vircadia/Iamus/commit/d39b5731e0481cc0006dae2e6fb4a8530137b0bb">view &bull;</a> Update ChangeLog.md</li> 
```

Comment:
Why do these all have double bullet points?

Reply:
They are generated using a script in the Iamus repo I believe. Just a way to format it I suppose, I copy and paste it over from here https://github.com/vircadia/Iamus/blob/master/ChangeLog.md.